src/components/TextInput/BaseTextInput.js

```diff
@@ -264,6 +264,8 @@ function BaseTextInput(props) {
 
                             // When autoGrow is on and minWidth is not supplied, add a minWidth to allow the input to be focusable.
                             props.autoGrow && !textInputContainerStyles.minWidth && styles.mnw2,
+                            // Remove border bottom when field is not editable.
+                            !isEditable && styles.borderNone,
```

Comment:
if you have time, could provide screenshots that shows the difference when we have this line and not?

Reply:
Rather than messing with the text input component, is there a way to reuse the global styles we're using for this elsewhere, like the profile page?

<img width="378" alt="image" src="https://github.com/Expensify/App/assets/2319350/86dc9258-860f-4b88-aecc-eccd5432ba83">